### PR TITLE
Implementing test_format_intervention_message

### PR DESCRIPTION
**Pull request by dantaeyoung**
Implementing test_format_intervention message and fixing some typos in banneduser_experiment_test.yml

### CODE REVIEW

**Comment by dantaeyoung** [comment]
@wllrd: Question: Should I remove 'thing_id' from the test parameters and hard-code it into the logic of the method, since it's arbitrary and not relevant to the message format at hand?

**Comment by wllrd** [comment]
> @wllrd: Question: Should I remove 'thing_id' from the test parameters and hard-code it into the logic of the method

yes, I'd just hardcode the ID value (rather than using a parameter) because it is not being tested.

**Comment by dantaeyoung** [comment]
@wllrd Ok, done

File: config/experiments/banneduser_experiment_test.yml [hunk 1]
```diff
@@ -8,14 +8,14 @@ test:
       next_randomization: 0
       arms:
         arm_0:
-          pm_subject: PM Subject Line (Newcomber Arm 0)
+          pm_subject: PM Subject Line (Newcomer Arm 0)
```
Comment (wllrd) [inline config/experiments/banneduser_experiment_test.yml:11]: this experiment don't comb.

**Review by wllrd — APPROVED** [review]
new test passes for me; lookin' good!

**Comment by dantaeyoung** [comment]
Closing issue #53

**Pull request merged**